packages/repository/src/model.ts

```diff
@@ -145,6 +149,63 @@ export class ModelDefinition {
     return this;
   }
 
+  /**
+   * Define a new "belongs to" relation.
+   * @param name - The name of the "belongs to" relation.
+   * @param definition - The definition for the "belongs to" relation.
+   */
+  belongsTo(
+    name: string,
+    definition: Partial<BelongsToDefinition> &
+      Pick<BelongsToDefinition, 'source' | 'target'>,
```

Comment:
I find it a bit difficult to understand which properties are required and which are optional. Relation definitions have most properties optional, do we need to apply `Partial` here?

It makes me wonder if something like the following would work too:

```ts
definition: Omit<BelongsToDefinition, 'name' | 'type' | 'targetsMany'>,
```

Feel free to ignore this comment if my suggestion requires too many changes to make it work.

Reply:
I agree this is a lot more clear. I will update the PR soon.